blocks/cards/cards.css

```diff
@@ -1,27 +1,64 @@
-.cards > ul {
+ .cards > ul {
   list-style: none;
-  margin: 0;
-  padding: 0;
   display: grid;
-  grid-template-columns: repeat(auto-fill, minmax(257px, 1fr));
-  grid-gap: 24px;
+  grid-template-columns: 1fr; /* Default: single column for mobile */
+  grid-gap: 16px; /* Space between cards */
+} 
+
+.cards > ul > li{
+  position: relative;
+  margin:20px;
+}
+
+.cards > ul > li >p{
+  margin:0;
 }
 
-.cards > ul > li {
-  border: 1px solid #dadada;
-  background-color: var(--background-color);
+.cards-card-para{
+display: block;
+} 
+
+.cards-card-para.hidden {
+  display: none;
 }
 
-.cards .cards-card-body {
-  margin: 16px;
+.cards-card-body { /* Match your image bg color */
+text-align: center;
+background: #484646;
+padding: 80px 40px 40px;
+  color: #fff;
 }
 
-.cards .cards-card-image {
+ .cards .cards-card-image {
+  margin-bottom: -70px; 
+  margin-left: 10px;
+  margin-right: 10px;
   line-height: 0;
+  display: block;
 }
 
-.cards > ul > li img {
+.cards-card-image img {
   width: 100%;
+  display: block;
   aspect-ratio: 4 / 3;
   object-fit: cover;
 }
+
+.report-button{
+    padding: 13.8px 15px;
+    font-family: "Open Sans SemiBold", "Open Sans", sans-serif;
+    text-align: left;
+    display: inline-block;
+    border-radius: 0;
+    color: white;
+    background: #004E9A; 
+
+     --link-hover: none;
+     --link-hover-color: #004E9A;
```

Comment:
variables need to be created only in styles.css, and not in block level css files. 

Flow generally is:
create variables in styles.css and use in block level css files.